src/MoBi.Core/Serialization/Converter/v3_5/Converter341To351.cs

```diff
@@ -93,14 +93,17 @@ public  int ConvertXml(XElement element, IMoBiProject project)
          //only happens when loading an actual MoBiSimulation from a project
          _loadingMoBiSimulation = (element.Name == AppConstants.XmlNames.MoBiSimulation);
 
-         return ProjectVersions.V3_5_1;
+         //too many conversion happening here! set to true for compatibility
+         return (ProjectVersions.V3_5_1, true);
```

Comment:
setting to true here is just a simple way to be compatible. Better safe than sorry